src/pages/preferences/preferencesTabs/notificationTab/ClearSubscriptions.tsx

```diff
@@ -23,9 +24,10 @@ export const ClearSubscriptions: React.FC = () => {
     onCompleted: (result) => {
       setShowModal(false);
       dispatchToast.success(
-        `Successfully cleared ${result.clearMySubscriptions} subscription${
-          result.clearMySubscriptions !== 1 && "s"
-        }!`
+        `Successfully cleared ${result.clearMySubscriptions} ${pluralize(
+          "subscription",
+          result.clearMySubscriptions
+        )}!`
```

Comment:
On second thought I feel like the `!` in "Success!" is enthusiastic enough
```suggestion
        )}`
```